client/src/app/site/projector/components/projector-detail/projector-detail.component.ts

```diff
@@ -353,4 +350,16 @@ export class ProjectorDetailComponent extends BaseViewComponentDirective impleme
             this.messageRepo.create(sendData).then(() => {}, this.raiseError);
         }
     }
+
+    private async loadSubscriptions(): Promise<void> {
+        await this.stableService.isStable;
+        this.countdownRepo.getViewModelListObservable().subscribe(countdowns => (this.countdowns = countdowns));
+        this.messageRepo.getViewModelListObservable().subscribe(messages => (this.messages = messages));
+        this.repo.getViewModelListObservable().subscribe(projectors => (this.projectorCount = projectors.length));
+    }
+
+    private async installUpdater(): Promise<void> {
+        await this.stableService.isStable;
+        this.subscriptions.push(timer(0, 500).subscribe(() => this.cd.detectChanges()));
```

Comment:
This seems hacky. Why do you need a detect changes 500ms after the app stable event? Please put the `detectChanges` to the place, where it is actually needed.

As a sidenote: You can use https://github.com/OpenSlides/OpenSlides/blob/master/client/src/app/core/promises/sleep.ts: `await SleepPromise(500)` if you want to do that kind of waiting.